src/remote_adapter/remote_adapter_client/qt_impl/lua_lib/sdl_remote_adapter_qt_client_lua_wrapper.cc

```diff
@@ -0,0 +1,199 @@
+#include "qt_impl/lua_lib/sdl_remote_adapter_qt_client_lua_wrapper.h"
+
+#include <iostream>
+
+#include "qt_impl/sdl_remote_adapter_qt_client.h"
+
+namespace lua_lib {
+
+int SDLRemoteTestAdapterLuaWrapper::create_SDLRemoteTestAdapter(lua_State* L) {
+  luaL_checktype(L, 1, LUA_TTABLE);
+
+  // Index -1(top) - table out_params
+  // Index -2 - table in_params
+  // Index -3 - number port
+  // Index -4 - string host
+  // index -5 - Library table
+
+  auto out_mq_params = build_MqParams(L);
```

Comment:
@aderiabin may be function build_MqParams will remove value from the top of the stack and used in lua library

Reply:
@LuxoftAKutsan It is not principal. Current decision was chosen to simplify understanding of Lua stack manipulations and unify Lua stack manipulation approach (get data + remove data from stack ) :
```
  // Get data from Lua stack
  auto port = lua_tointeger(L, -1); 
  auto ip = lua_tostring(L, -2);
  // Remove 2 values from top of the stack
  lua_pop(L, 2);